packages/zwave-js/src/lib/telemetry/metrics.ts

```diff
@@ -0,0 +1,90 @@
+import { formatId } from "@zwave-js/shared";
+import { isObject } from "alcalzone-shared/typeguards";
+import axios from "axios";
+import * as crypto from "crypto";
+import type { Driver } from "../driver/Driver";
+
+const baseUrl = "https://metrics.zwave-js.io:9998"; // No leading slash!
+// The shared "secret" between metrics server and zwave-js. Can be used to disable reporting for misbehaving clients.
+const authKey = "Og9O}$[wKk=<$Z*qUduK";
```

Comment:
how? why not use the application version since changing this would require a release?

Reply:
We could use an api to provide the auth key IMO. Yeah changing this in this way could break old instances instantly

Reply:
an api to provide an authkey to an api?
seems like unnecessary calls and just something else to get clobbered.
hash of the homeid should be enough to track and block bad clients that aren't out to get you

Reply:
failing that a WAF might not be a bad idea

Reply:
Like we discussed earlier - this should rather be a "client id" or "schema id" or "token". Just a simple publicly known value that can easily be changed in a config file and gets used to ignore every request by a client that's not on the latest version.

This does not mean that specific implementations shouldn't do some on-top validation serverside.